rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -597,11 +606,9 @@ public String externalIp() {
             tryParseIpOrThrow();
             logger.info("External address identified: {}", externalIp);
         } catch (IOException e) {
-            externalIp = bindIp();
-            logger.warn("Can't get external IP. Fall back to peer.bind.ip: " + externalIp + " :" + e);
+            logger.warn("Can't get external IP. " + e);
```

Comment:
Just asking. What behavior we expect when it fails?

Reply:
I already modified the logger, to log an error instead of a warning.
If no external ip is found, the node is going to log the error and an exception but it will continue the normal process.
A case where the node has no external ip could be in some testing environment as standalone or in a closed network without internet access.